pkg/branding/env.go

```diff
@@ -0,0 +1,17 @@
+package branding
+
+import "github.com/stackrox/rox/pkg/env"
+
+const (
+	// ProductBranding should hold RHACS_BRANDING or STACKROX_BRANDING
+	ProductBrandingEnvName = "ROX_PRODUCT_BRANDING"
+)
+
+var (
+	productBrandingSetting = env.RegisterSetting(ProductBrandingEnvName, env.WithDefault("RHACS_BRANDING"))
```

Comment:
I discussed this with Fred and the idea was not to change current behavior as previously it was all RHACS branded. But what you say makes sense to me, I can just remove the default again then.

Reply:
Alternatively, if you plan following-up this with a change to the `Makefile` and Circle's `config.yml`, you may add a `// TODO` comment that would signal that the default should be removed.

Reply:
Changing it now, we will have to make sure that we deliver it to our RHACS customers with the value set correctly then.

Reply:
I agree that ultimately the default should be OpenSource. My point was that if we're splitting this into two tasks, once this PR is merged we can't take for granted that the follow-up will be merged before the next release is cut. To avoid commercial customers seeing the StackRox branding, it's better to only switch this on the follow-up PR, whenever that happens.

Reply:
Okay, I'll revert it to defaulting to `RHACS_BRANDING` and add a // TODO